src/qmflows/packages/packages.py

```diff
@@ -638,8 +638,6 @@ def import_parser(ds: Mapping[str, str], module_root: str = "qmflows.parsers") -
 def find_file_pattern(path: Union[str, os.PathLike],
                       folder: Union[None, str, os.PathLike] = None) -> Iterator[str]:
     if folder is not None and os.path.exists(folder):
-        print("folder: ", folder)
```

Comment:
I assumed these were leftover ``print()`` calls from some debugging.
If they're required then I can just add a ``Logger.info()`` call back here.

Reply:
Indeed! thanks for removing them :+1: